extensions/issuance/local-statuslist-publisher/src/main/java/org/eclipse/edc/issuerservice/publisher/LocalStatusListCredentialPublisherExtension.java

```diff
@@ -61,9 +72,10 @@ public String name() {
 
     @Override
     public void initialize(ServiceExtensionContext context) {
+        var objectMapper = typeManager.getMapper(JSON_LD);
         portMappingRegistry.register(new PortMapping(STATUS_LIST, config.port(), config.path()));
 
-        webServer.registerResource(STATUS_LIST, new StatusListCredentialController(store, context.getMonitor(), new ObjectMapper()));
+        webServer.registerResource(STATUS_LIST, new StatusListCredentialController(store, monitor, objectMapper));
```

Comment:
I think it's better to pass or a `Supplier<ObjectMapper>` or the type manager and the and the `JSON_LD` key to the `StatusListCredentialController`. We had strange behavior in the past where the `JSON_LD`  was not registered yet in the `TypeManager` and the get returned the default one, which probably it's not the expected one